newspaper/text.py

```diff
@@ -182,3 +182,29 @@ def get_stopword_count(self, content):
         ws.set_stopword_count(len(overlapping_stopwords))
         ws.set_stop_words(overlapping_stopwords)
         return ws
+
+
+class StopWordsNepali(StopWords):
```

Comment:
Notice that `StopWordsNepali` has identical code with the inherited `Stopwords`. Please correct me if wrong but we don't need the new `StopWordsNepali` class, we can just re-use `Stopwords` for the `np` language